testsuite/features/step_definitions/common_steps.rb

```diff
@@ -103,7 +103,7 @@
     step %(I should see a "#{rounded_uptime_hours} hours ago" text)
   elsif rounded_uptime_minutes >= 45 && rounded_uptime_hours == 1 # shows "an hour ago" from 45 minutes onwards up to 1.5 hours
     step %(I should see a "an hour ago" text)
-  elsif rounded_uptime_minutes > 1 && rounded_uptime_hours < 1
+  elsif rounded_uptime_minutes > 1 && rounded_uptime_hours <= 1
```

Comment:
How that is related to enabling Hardware refresh? If that's out of topic, must go in a different PR

Reply:
This makes the test succeed :)  . If you have an uptime of 35 minutes, when you rounded is 1h. Without this fix, the test was looking for 0 days instead.

Reply:
Please remove this change then here once the other PR is merged.